components/navbar/navbar.css

```diff
@@ -55,8 +55,7 @@
 }
 
 .ampstart-navbar-trigger {
-  line-height: var(--navbar-height);
-  font-size: var(--h2);
+  height: 24px;
```

Comment:
i think  this is probably because the char was not unicode or utf8 , if we find the right char this should be solve-able easily